docs/kpod-push.1.md

```diff
@@ -113,5 +117,17 @@ Writing manifest to image destination
 Storing signatures
 ```
 
+This example pushes the alpine to test-dir with the "oci" manifest type
+```
+kpod push --format oci alpine dir:test-dir
```

Comment:
oh...mmm

Reply:
Use an example using one of our operating systems.  rhel7?

Reply:
fixed